bigchaindb/common/schema/transaction_vote_v2.0.yaml

```diff
@@ -5,14 +5,14 @@
 ---
 "$schema": "http://json-schema.org/draft-04/schema#"
 type: object
-title: Validator Election Vote Schema - Vote on a validator set change
+title: Vote Schema - Vote on an election
 required:
 - operation
 - outputs
 properties:
   operation:
     type: string
-    value: "VALIDATOR_ELECTION_VOTE"
+    value: "OPERATION"
```

Comment:
Value is supposed to be `VOTE`

Reply:
It is weird that the CI tests are still passing even when the election created using `.generate` would not match the above schema!

Reply:
done

Reply:
but the value is still `"OPERATION"`?